src/manifolds/ShapeSpace.jl

```diff
@@ -0,0 +1,241 @@
+
+@doc raw"""
+    KendallsPreShapeSpace{n,k} <: AbstractSphere{ℝ}
```

Comment:
Maybe we should document here that the is a quotient manifold (maybe mentioning traits in general would be cool in the manifold docs).

Reply:
I don't really treat the pre-shape space for the purpose of implementation as a quotient though it is sometimes defined as such, I will mention it here.

Reply:
Ok, might be useful in the future to add that as well, but maybe too much for this PR.